.azure/pipelines/jobs/default-build.yml

```diff
@@ -97,7 +97,7 @@ jobs:
         vmImage: macOS-11
       ${{ if eq(parameters.agentOs, 'Linux') }}:
         ${{ if and(eq(parameters.useHostedUbuntu, true), or(ne(variables['System.TeamProject'], 'internal'), in(variables['Build.Reason'], 'Manual', 'PullRequest', 'Schedule'))) }}:
-          vmImage: ubuntu-18.04
+          vmImage: ubuntu-20.04
```

Comment:
@MattGal a couple of lines below this, we're using `Build.Ubuntu.1804.Amd64[.Open]` image overrides. I'd like to switch to the equivalent Ubuntu 20.04 image. Is that `1es-ubuntu-2004[.-Open]`❔ (Not going to try using that if a new `Build.Ubuntu.2004.Amd64[.Open]` is the way to go.)

Reply:
> @MattGal a couple of lines below this, we're using `Build.Ubuntu.1804.Amd64[.Open]` image overrides. I'd like to switch to the equivalent Ubuntu 20.04 image. Is that `1es-ubuntu-2004[.-Open]`❔ (Not going to try using that if a new `Build.Ubuntu.2004.Amd64[.Open]` is the way to go.)

This is primarily a consideration of disk space usage, especially so if you build in docker containers.  If your build needs <= 10 GB free, I'd always favor the [1es-ubuntu-2004](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md) since it gets updated automatically in approximate synch with what our users are using publicly in their CI systems.  If you run out of disk space, use the `build.ubuntu` images to get ~100 GB free space on the workspace drive (it uses the VM's ephemeral disk for the workspace).

Reply:
Oops.  We also don't have build.ubuntu 20.04 yet :) I'll get an issue created.

Reply:
Created https://github.com/dotnet/arcade/issues/10833 to track getting this and 22.04 images stood up.